PIVX-Labs/MyPIVXWallet
# Auto detect currency

## Abstract

Choose the correct currency based on user locale.
Closes #250

## Testing
- Open MPW in a private window (this only works on first load)
- Check that the currency matches your current locale (en-US = USD, en-GB = GBP, de-DE = EUR, ...)
